core/src/Kokkos_Core_fwd.hpp

```diff
@@ -324,6 +324,9 @@ template <class FunctorType, class ExecPolicy, class ReturnType = InvalidType,
               FunctorType, ExecPolicy>::execution_space>
 class ParallelScanWithTotal;
 
+/** \brief Get GPU assigned by CTest */
+int get_ctest_gpu(const char* local_rank_str);
```

Comment:
Please provide a rational for exposing `get_ctest_gpu()` or drop it and add a declaration in the unit test.

Reply:
I will move the declaration to the unit test